src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -104,24 +90,23 @@ class DraftJSEditor extends React.Component<Props> {
 
         return (
             <div className={classes}>
-                <Label hideLabel={hideLabel} showOptionalText={!isRequired} text={label}>
-                    <Tooltip isShown={!!error} position="bottom-left" text={error ? error.message : ''} theme="error">
-                        {/* need div so tooltip can set aria-describedby */}
-                        <div>
-                            <Editor
-                                {...a11yProps}
-                                editorState={editorState}
-                                handleReturn={this.handleReturn}
-                                onBlur={handleBlur}
-                                onChange={handleChange}
-                                onFocus={onFocus}
-                                placeholder={placeholder}
-                                readOnly={isDisabled}
-                                stripPastedStyles
-                            />
-                        </div>
-                    </Tooltip>
-                </Label>
+                <Tooltip isShown={!!error} position="bottom-left" text={error ? error.message : ''} theme="error">
+                    {/* need div so tooltip can set aria-describedby */}
+                    <div>
+                        <Editor
+                            {...a11yProps}
+                            ariaLabel={ariaLabel === undefined ? placeholder : ariaLabel}
```

Comment:
How about `ariaLabel || placeholder`? In addition, `placeholder` is not a required prop neither, you may consider add a default prop value for one of them.

Reply:
Actually I do not think these two should be associated w/ each other. Now setting ariaLabel={ariaLabel} @mxiao6